src/Compilers/Test/Utilities/CSharp/CompilationTestUtils.cs

```diff
@@ -278,5 +278,76 @@ public static List<string> LookupNames(this SemanticModel model, int position, I
                     : model.LookupSymbols(position, container);
             return symbols.Select(s => s.Name).Distinct().ToList();
         }
+
+        internal static void VerifyTypes(this CSharpCompilation compilation)
+        {
+            foreach (var tree in compilation.SyntaxTrees)
+            {
+                VerifyTypes(compilation, tree);
+            }
+        }
+
+        internal static void VerifyTypes(this CSharpCompilation compilation, SyntaxTree tree)
```

Comment:
>VerifyTypes [](start = 29, length = 11)

Please add a doc comment with clear description what is being verified and detailed description of conventions used.